pkg/oc/cli/cmd/startbuild.go

```diff
@@ -105,6 +105,8 @@ func NewCmdStartBuild(fullName string, f *clientcmd.Factory, in io.Reader, out,
 
 	cmd.Flags().BoolVarP(&o.Follow, "follow", "F", o.Follow, "Start a build and watch its logs until it completes or fails")
 	cmd.Flags().BoolVarP(&o.WaitForComplete, "wait", "w", o.WaitForComplete, "Wait for a build to complete and exit with a non-zero return code if the build fails")
+	cmd.Flags().BoolVar(&o.Incremental, "incremental", o.Incremental, "Override incremental")
```

Comment:
think we need to elaborate with the help at least a little ... explain what api obj the flag is on, or which strategy it applies to, or the behavior it controls